docs/docs/Components/components-agents.md

```diff
@@ -45,37 +48,99 @@ For more information on this component, see the [tool calling agent documentatio
 | handle_parsing_errors | Boolean | Whether to handle parsing errors during agent execution. |
 | verbose | Boolean | Enables verbose output for detailed logging. |
 
-### Outputs
+**Outputs**
 
 | Name | Type | Description |
 |------|------|-------------|
 | response | Message | The agent's response to the given input task. |
 
-## CSV Agent
+</details>
 
-This component creates a CSV agent from a CSV file and LLM.
+## Legacy components
+
+Legacy components are available to use but no longer supported.
```

Comment:
Replace with "Legacy components are available for use but are no longer supported." 

Apply this update to all instances of this sentence.